### PR TITLE
[Merged by Bors] - feat(fvm): set version as active after install

### PR DESCRIPTION
**Pull request by EstebanBorai**
`fvm install` now sets the installed version as active

---

## Demo

https://github.com/infinyon/fluvio/assets/34756077/4aea12c9-dd0d-46ee-9780-483036e9abf7

### CODE REVIEW

**Comment by EstebanBorai** [comment]
@digikata I just cherry-picked your commit 5e5c45cc5f77018e8187cfa4917a8e9067fb37d0 here!

File: tests/cli/fvm_smoke_tests/fvm_basic.bats [hunk 1]
```diff
@@ -504,3 +511,35 @@ setup_file() {
...
+
+    # Sets `fvm` in the PATH using the "env" file included in the installation
+    source ~/.fvm/env
+
+    run bash -c 'fvm current'
+    assert_line --index 0 "warn: No active version set"
+    assert_line --index 1 "help: You can use fvm switch to set the active version"
+    assert_success
+
+    run bash -c 'fvm install 0.10.14'
```
Comment (sehz) [inline tests/cli/fvm_smoke_tests/fvm_basic.bats:527]: Can you put version as variable?

**Review by digikata — APPROVED** [review]
LGTM

**Review by sehz — CHANGES_REQUESTED** [review]
Just nit

File: tests/cli/fvm_smoke_tests/fvm_basic.bats (hunk 1 above)
Comment (EstebanBorai) [reply tests/cli/fvm_smoke_tests/fvm_basic.bats]: Fixed here: [a1d277f](https://github.com/infinyon/fluvio/pull/3604/commits/a1d277f6cd3252361e6b6de0ac9ce4d4705bbef5)

**Review by sehz — APPROVED** [review]
LGTM

**Pull request closed without merging**